.github/workflows/rebuild-push-dist.yml

```diff
@@ -0,0 +1,62 @@
+name: Dependabot post-update
+on:
+  push:
+    branches:
+      - dependabot/npm_and_yarn/**

```

Comment:
decided to go for branch instead of PR because it would allows for easier pushing and no need to checkout a sha reference for PR and trying to resolve branch ref.

Reply:
I makes sense, thanks for the explaination